<launch>
  <arg name="port" default="/dev/ttyUSB0"/>

  <node name="standalone_driver" pkg="gps_driver" type="standalone_driver.py" output="screen">
    <param name="port" value="$(arg port)"/>
  </node>
</launch>
